wpilib/src/lib.rs

```diff
@@ -4,11 +4,24 @@
 // <https://www.gnu.org/licenses/> for a copy.
 
 #![feature(concat_idents)]
-
-#[macro_use]
+#![macro_use]
```

Comment:
Hang on. Why was this line changed? You want to use the `lazy_static!` macro here, right?

Reply:
It did not compile without the exclamation mark.